style.css

```diff
@@ -0,0 +1,39 @@
+body {
```

Comment:
Great job on keeping your .css file DRY and following its best practices!

Reply:
😊😊 Thanks for your comments!